src/MainWindow.vala

```diff
@@ -3,61 +3,57 @@
  * SPDX-FileCopyrightText: 2021 elementary, Inc. (https://elementary.io)
  */
 
-public class Music.MainWindow : Hdy.ApplicationWindow {
+public class Music.MainWindow : Gtk.ApplicationWindow {
     construct {
-        Hdy.init ();
+        // try {
+        //     var css_provider = new Gtk.CssProvider ();
+        //     css_provider.load_from_data ("@define-color accent_color @ORANGE_500;");
 
-        try {
-            var css_provider = new Gtk.CssProvider ();
-            css_provider.load_from_data ("@define-color accent_color @ORANGE_500;");
+        //     Gtk.StyleContext.add_provider_for_display (Gdk.Display.get_default (), css_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+        // } catch (Error e) {
+        //     critical ("Can't set accent color: %s", e.message);
+        // }
 
-            Gtk.StyleContext.add_provider_for_screen (Gdk.Screen.get_default (), css_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
-        } catch (Error e) {
-            critical ("Can't set accent color: %s", e.message);
-        }
-
-        var queue_header = new Hdy.HeaderBar () {
+        var queue_header = new Adw.HeaderBar () {
+            css_classes = {"default-decoration", "flat"},
             hexpand = true,
-            show_close_button = true
+            show_end_title_buttons = false
         };
 
-        unowned var queue_header_context = queue_header.get_style_context ();
-        queue_header_context.add_class (Granite.STYLE_CLASS_DEFAULT_DECORATION);
-        queue_header_context.add_class (Gtk.STYLE_CLASS_FLAT);
-
-        var queue_placeholder = new Granite.Widgets.AlertView (
-            _("Queue is Empty"),
-            _("Audio files opened from Files will appear here"),
-            "playlist-queue"
-        );
-        queue_placeholder.show_all ();
+        // var queue_placeholder = new Granite.Widgets.AlertView (
+        //     _("Queue is Empty"),
+        //     _("Audio files opened from Files will appear here"),
+        //     "playlist-queue"
+        // );
 
         var queue_listbox = new Gtk.ListBox () {
-            expand = true
+            hexpand = true,
+            vexpand = true
         };
         queue_listbox.bind_model (PlaybackManager.get_default ().queue_liststore, create_queue_row);
-        queue_listbox.set_placeholder (queue_placeholder);
+        // queue_listbox.set_placeholder (queue_placeholder);
 
-        var scrolled = new Gtk.ScrolledWindow (null, null);
-        scrolled.add (queue_listbox);
+        var scrolled = new Gtk.ScrolledWindow () {
+            child = queue_listbox
```

Comment:
One annoying thing: the implicit GtkViewport gtk inserts here does _not_ have `scroll-to-focus=true` and you need to add one manually.